response_operations_social_ui/templates/social-change-response-status.html

```diff
@@ -5,7 +5,7 @@
 {%- block main %}
 <div role="main" id="main" class="page__main">
   <div class="spoke__header">
-    <div class="spoke__header__title">
+    <div class="spoke__header--title">
```

Comment:
Looking at the `scss` we could get rid of this element and just add the padding to the `spoke__header`. If this has to stay for some reason it's not quite `BEM` as it's not a modifier and should be something like `spoke__title`.

Reply:
Or add `spoke__title` to the `h1` and get rid of the `div`